genai-perf/genai_perf/parser.py

```diff
@@ -246,6 +246,49 @@ def _check_load_manager_args(args: argparse.Namespace) -> argparse.Namespace:
     return args
 
 
+def is_valid_url(url):
+    """
+    Checks if a URL is valid. It must use 'http' or 'https', have a valid
+    netloc, optional port, and contain '/metrics' in the path.
```

Comment:
can we add this to the docstring:
```
<scheme>://<netloc>/<path>;<params>?<query>#<fragment>
```
so we know what the pattern is for the url?